scripts/prepare-release.js

```diff
@@ -332,6 +333,55 @@ async function updateBackstageReleaseVersion(repo, type) {
   );
 }
 
+/**
+ * Ensures that the changesets include a version bump of create-app otherwise
+ * generates a new patch changeset for create-app.
+ */
+async function ensureCreateAppChangeset() {
+  process.chdir(path.resolve(__dirname, '../.changeset'));
+
+  const fileNames = await fs.readdir('.');
+  const changesetNames = fileNames.filter(
+    name => name.endsWith('.md') && name !== 'README.md',
+  );
+
+  const changesets = await Promise.all(
+    changesetNames.map(async name => {
+      const content = await fs.readFile(name, 'utf8');
+      return { name, ...parseChangeset(content) };
+    }),
+  );
+
+  let excludeList = [];
+  if (await fs.pathExists('pre.json')) {
+    const data = await fs.readJSON('pre.json');
+    excludeList = data.changesets.map(name => `${name}.md`);
```

Comment:
Except if `pre.json` is in `mode: "exit"`, in which case I don't think we should exclude any of the changesets